packages/editor/src/package/storage-format.ts

```diff
@@ -35,6 +59,7 @@ const migrations: Migration[] = [
     return {
       ...state,
       editorVersion,
+      domainOrigin: window.location.origin,
```

Comment:
If the package is already deployed to production (could be the case in RLP, not sure) we should not change existing migrations. But instead we can create a new migration at the end of the `migrations` array.

Reply:
Hm, I prefer a separate migration. See https://github.com/serlo/frontend/pull/4070